source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -123,18 +133,21 @@ private void bitsComparison() {
 
     private void geoWithinComparison() {
         // begin geoWithinComparison
+        // Creates a Polygon to filter the results
         Polygon square = new Polygon(Arrays.asList(new Position(0, 0), 
                                             new Position(4, 0), 
                                             new Position(4, 4), 
                                             new Position(0, 4),
                                             new Position(0, 0)));
    
+        // Matches documents with "coordinates" values that fall within the Polygon filter
         Bson geoWithinComparison = geoWithin("coordinates", square);
         collection.find(geoWithinComparison).forEach(doc -> System.out.println(doc.toJson()));
         // end geoWithinComparison
     }
 
     private void preview(){
+        // Uses an empty filter to print all documents in the collection
```

Comment:
Suggestion:
I think this should be omitted since it's setup code meant only for tech writers